commands/run.js

```diff
@@ -31,15 +31,14 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'run',
   description: 'run a one-off process inside a heroku dyno',
-  help: `
-Examples:
+  help: `#### Examples:
```

Comment:
we shouldn't add this kind of encoding unless we have a way to strip it back out I feel